src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -219,8 +219,7 @@ public ParseOptions ParseOptions
         /// <summary>
         /// The path to the output in obj.
         /// </summary>
-        /// <remarks>This is internal for now, as it's only consumed by <see cref="EditAndContinue.VsENCRebuildableProjectImpl"/>
-        /// which directly takes a <see cref="VisualStudioProject"/>.</remarks>
+        /// <remarks>TODO: remove, see https://github.com/dotnet/roslyn/issues/34371</remarks>
```

Comment:
📝 #34371 to force a direct link from the PR 😄 

❔ Would it help to go one step further and mark this `[Obsolete]`?

Reply:
Not sure what this comment applies to. CodeFlow doesn't align it with anything and I cna't see it on github :(

---
In reply to: [270625556](https://github.com/dotnet/roslyn/pull/34027#discussion_r270625556) [](ancestors = 270625556)